mapie/tests/test_classification.py

```diff
@@ -528,7 +528,7 @@ def predict_proba(self, X: ArrayLike) -> NDArray:
             )
         else:
             return np.array(
-                [[0.2, 0.7, 0.1], [0.1, 0.2, 0.7], [0.3, 0.5, 0.2]]
+                [[0.2, 0.7, 0.1], [0., 1., 0.], [0., .7, 0.3]]
```

Comment:
Add also maybe [0.3, 0.7, 0.], just to make sure it works in all cases.

Reply:
done